sema4ai/src/sema4ai_code/workspace_manager.py

```diff
@@ -120,25 +123,66 @@ def _get_agent_package_organizations(
             if not org_directory.is_dir():
                 continue
 
+            action_packages: DefaultDict[
+                str, List[Tuple[str, LocalAgentPackageOrganizationInfoDict]]
```

Comment:
As we're on 3.10, usually prefer `list[tuple[str, ...` instead of `List`, `Tuple`, etc.